.github/workflows/build-prs.yml

```
@@ -84,6 +84,8 @@ jobs:
             dockerfile: backend/test/images/Dockerfile.test
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)